prisma/schema.prisma

```diff
@@ -11,80 +8,77 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-// companies
 model Company {
   id       Int        @id @default(autoincrement())
   name     String     @unique
-  Rooms    Room[]
+  domain   String
```

Comment:
Looks fine, but should we maybe have an array of acceptable domains or just the one domain?

Reply:
yeah, that's a good idea. will change it to that